modules/nodes-clusters-cgroups-2.adoc

```diff
@@ -35,9 +35,14 @@ Currently, disabling CPU load balancing is not supported by cgroup v2. As a resu
 * You have a running {product-title} cluster that uses version 4.12 or later.
 * You are logged in to the cluster as a user with administrative privileges.
 
+[NOTE]
+====
+If you are installing an {product-title} cluster that uses single-root I/O virtualization (SR-IOV) or Open vSwitch with the Data Plane Development Kit (OVS-DPDK) on {rh-openstack-first}, you must understand the requirements for each technology and then perform preparatory tasks. For more information, see xref:../../installing/index.adoc#installing-openstack-nfv-preparing[Preparing to install a cluster that uses SR-IOV or OVS-DPDK on OpenStack].
```

Comment:
@DCChadwick, you [can't use an xref in a module](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references). I agree with Emilien - it makes more sense to update the shiftstack docs. Please include any xrefs you add in additional resources sections in the assembly. You can find examples throughout the collection, and the repo guidelines have information about format and of additional resource sections starting around [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies).

Reply:
@kalexand-rh Thank you!